ecosystem/sep-0023.md

```diff
@@ -77,14 +79,20 @@ The following steps transform a binary key into a strkey:
 3.  If we are encoding a multiplexed address, append an 8-byte memo ID
     in network byte order (most significant byte first).
 
-4.  Compute a 16-bit CRC16 checksum of the combined version byte,
+4.  If we are encoding a signed payload, append a 4-byte length in
+    network byte order (most significant byte first) that holds the
+    length of the payload, then append the payload, and finally zero
+    padding of 0 to 3 zero bytes such that the total number of bytes of
+    the payload plus the zero padding is a multiple of four.
+
+5.  Compute a 16-bit CRC16 checksum of the combined version byte,
     optional memo ID, and binary key (using polynomial
```

Comment:
optional memo ID, optional payload, and binary key

Reply:
817c10f6cd06bc48daa9e59f6f284207d74b7804